lib/koans/03_sigils.ex

```diff
@@ -0,0 +1,39 @@
+defmodule Sigils do
+  use Koans
+
+  @intro "Sigils"
+
+  koan "The ~s sigil is a different way of expressing string literals" do
+    assert ~s{This is a string} == ___
+  end
+
+  koan "Sigils are useful to avoid escaping quotes in strings" do
+    assert "\"Welcome to the jungle\", they said." == ___
+  end
+
+  koan "Sigils can use different delimiters" do
+    matches? = ~s{This works!} == ~s[This works!]
+    assert matches? == ___
+  end
+
+  koan "The lowercase ~s sigil supports string interpolation" do
+    assert ~s[1 + 1 = #{1+1}] == ___
+  end
+
+  koan "The ~S sigil is similar to ~s but doesn't do interpolation" do
+    assert ~S[1 + 1 = #{1+1}] == ___
+  end
+
+  koan "The ~w sigil creates word lists" do
+    assert ~w(Hello world) == [___, ___]
```

Comment:
Isn't the fact that a list is being created obvious enough from the title?
I have gut feeling that this is 'too easy' 😅